website/content/en/blog/tracking-allocations.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Monitoring Vector's per-component memory usage
+short: Allocation tracking
+description: Gaining insight into Vector's per-component memory usage.
+authors: ["arshiyasolei", "tobz"]
+date: "2022-12-13"
+badges:
+  type: announcement
+  domains: ["monitoring"]
+tags: ["allocation-tracing", "tracking allocations"]
+---
+
+We are excited to announce that Vector now has support for exposing per-component memory usage metrics. This work addresses an often-requested feature from users who want to understand how Vector uses memory, and what parts of their configurations are responsible for high memory usage.
+
+## Trying it out
+
+![vector top with allocation tracing](/img/blog/vector-top-allocation-tracking.png)
+
+To explore the feature, start a Vector instance with the `allocation-tracing` option, launch `vector top` and monitor your components!
```

Comment:
Minor point, but the `api` is disabled by default so `vector top` would fail without that config.

I'd also consider splitting this up to be something like:
---
To explore this feature, start a Vector instance with the `allocation-tracing` option:

```shell
vector --allocation-tracing -c /foo/config.yaml
```

Then, launch `vector top` and monitor your components!
---

Not personally familiar with the feature, I don't know if the option is a flag or in the config file...

Reply:
Yeah the instructions do assume some familiarity with vector top